aiomusiccast/capability_registry.py

```diff
@@ -0,0 +1,260 @@
+from .features import ZoneFeature, DeviceFeature
+from .capabilities import OptionSetter, EntityTypes, NumberSetter, BinarySetter, Capability
+
+"""Dictionary of all DeviceFeatures with a callable as value. 
+The callable expects an ID and a MusicCastDevice as parameters."""
+_device_capabilities = {
+    DeviceFeature.DIMMER: lambda capability_id, device: NumberSetter(
+        capability_id,
+        "Display Brightness",
+        EntityTypes.CONFIG,
+        lambda: device.data.dimmer.dimmer_current,
+        lambda value: device.set_dimmer(int(value)),
+        device.data.dimmer.minimum,
+        device.data.dimmer.maximum,
+        device.data.dimmer.step,
+    ),
+    DeviceFeature.SPEAKER_A: lambda capability_id, device: BinarySetter(
+        capability_id,
+        "Speaker A",
+        EntityTypes.CONFIG,
+        lambda: device.data.speaker_a,
+        lambda val: device.set_speaker_a(val),
+    ),
+    DeviceFeature.SPEAKER_B: lambda capability_id, device: BinarySetter(
+        capability_id,
+        "Speaker B",
+        EntityTypes.CONFIG,
+        lambda: device.data.speaker_b,
+        lambda val: device.set_speaker_b(val),
+    ),
+    DeviceFeature.PARTY_MODE: lambda capability_id, device: BinarySetter(
+        capability_id,
+        "Party Mode",
+        EntityTypes.CONFIG,
+        lambda: device.data.party_enable,
+        lambda val: device.set_party_mode(val),
+    ),
+}
+
+
+"""Dictionary of all ZoneFeatures with a callable as value. 
+The callable expects an ID, a MusicCastDevice and a zone_id as parameters."""
+_zone_capabilities = {
+    ZoneFeature.SURR_DECODER_TYPE: lambda capability_id, device, zone_id: OptionSetter(
+        capability_id,
+        "Surround Decoder Device",
+        EntityTypes.CONFIG,
+        lambda: device.data.zones[zone_id].surr_decoder_type,
+        lambda val: device.set_surround_decoder(zone_id, val),
+        {key: key for key in device.data.zones[zone_id].surr_decoder_type_list},
+    ),
+    ZoneFeature.SLEEP: lambda capability_id, device, zone_id: OptionSetter(
+        capability_id,
+        "Sleep Timer",
+        EntityTypes.CONFIG,
+        lambda: device.data.zones[zone_id].sleep_time,
+        lambda val: device.set_sleep_timer(zone_id, val),
+        {0: "off", 30: "30 min", 60: "60 min", 90: "90 min", 120: "120 min"},
+    ),
+    ZoneFeature.EQUALIZER: {
+        "mode": lambda capability_id, device, zone_id: OptionSetter(
+            capability_id,
+            "Equalizer Mode",
+            EntityTypes.CONFIG,
+            lambda: device.data.zones[zone_id].equalizer_mode,
+            lambda val: device.set_equalizer(zone_id, mode=val),
+            {key: key for key in device.data.zones[zone_id].equalizer_mode_list},
+        ),
+        "low": lambda capability_id, device, zone_id: NumberSetter(
+            capability_id,
+            "Low",
```

Comment:
Maybe we should call this _Equalizer Low_ to make it more clear what this does. Same for mid and high

Reply:
good idea, I did will do it the same way for tone control